spec/lucky/error_handling_spec.cr

```diff
@@ -121,14 +123,16 @@ end
 private def handle_error(
   format : Symbol = :html,
   show_debug_output : Bool = false,
-  error : Exception = UnhandledError.new
+  error : Exception = UnhandledError.new,
+  status_code : Int32 = 200
```

Comment:
I believe that the default is not used in any of the specs so can be removed. But I may have overlooked something so if it’s used it is of course ok to leave as is :)

Reply:
Yeah, the default is still used in a couple examples:
https://github.com/luckyframework/lucky/blob/4fd49c186fba7f5d3f08d8acea5685a725f6dc9b/spec/lucky/error_handling_spec.cr#L36-L55

Reply:
Perfect. I’ll go ahead and merge it